5-unexpected-files/results.md

```diff
@@ -3,12 +3,38 @@
 Commit your data here
 
 Original build file listing:
+/my-app
+├── .dockerignore
+├── Gemfile
+├── Gemfile.lock
+├── Rakefile
+├── app
+│   ├── .env
+│   └── .ruby-version
+├── entrypoint.sh
+└── tests
+    ├── out
+    │   └── first_test_log.txt
+    └── test_something.rb
 
 
 
 
 Better build file listing:
 
+/my-app
+├── .dockerignore
+├── Gemfile
+├── Gemfile.lock
+├── Rakefile
+├── app
+│   └── .ruby-version
+├── entrypoint.sh
+└── tests
+    ├── out
```

Comment:
For production purposes tests are likely to be left out. the `out` folder should always be excluded to avoid stray test results being shipped and observable